### PR TITLE
fix bug where could not re-run staramr

### PR DESCRIPTION
**Pull request by Takadonet**
Fixing [bug ](https://github.com/phac-nml/staramr/issues/118)

### CODE REVIEW

File: tools/staramr/staramr_search.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<tool id="staramr_search" name="staramr" version="@VERSION@">
+<tool id="staramr_search" name="staramr" version="@VERSION@+galaxy1">
```
Comment (apetkau) [inline tools/staramr/staramr_search.xml:1]: Is there a reason you updated the version like this? I thought for quick bug fixes in the wrapper we should leave the version alone (so anyone who installs the tool will get the latest version). But, I don't know if this has changed (I haven't made Galaxy tools in a while).

Comment (Takadonet) [reply tools/staramr/staramr_search.xml:1]: They have changed to the **+galaxy#** format a while ago but was not 100% sure if this case needs to have it. 

What do you think?

Comment (apetkau) [reply tools/staramr/staramr_search.xml:1]: I don't think it matters too much. So let's just stick with what you've submitted now. Looks like it passed so I'm merging.

Comment (bgruening) [reply tools/staramr/staramr_search.xml:1]: Hi, the version should be increased as soon as the results could be impacted. If you fix help text and such a version increase is not needed,

Comment (Takadonet) [reply tools/staramr/staramr_search.xml:1]: Thanks for the clarification!

Comment (apetkau) [reply tools/staramr/staramr_search.xml:1]: Yes. Thank you 👍

**Pull request merged**